src/DataFetcher.py

```diff
@@ -261,3 +265,32 @@ def parse_yahoo_finance_quote_summary(self, response, *args, **kwargs):
     success = self.yahoo_finance_quote_summary.parse_modules(result)
     if not success:
       self.yahoo_finance_quote_summary = None
+
+  def get_roic_averages(self):
+    """
+    Calculate ROIC averages for 1,3,5 and Max years
+    StockRow averages aren't accurate, so we're getting avgs for 1y and 3y from Yahoo
+    by calculating these by ouselves. The rest is from StockRow to at least have some (even
+    a bit inaccurate values), cause Yahoo has data for 4 years only.
+    """
+    roic_avgs = []
+    try:
+      roic_avgs.append(self.yahoo_finance_quote_summary.get_roic_average(years=1))
+    except AttributeError:
+      try:
+        roic_avgs.append(self.stockrow_key_stats.roic_averages[0])
+      except IndexError:
+        return []
+    try:
+      roic_avgs.append(self.yahoo_finance_quote_summary.get_roic_average(years=3))
+    except AttributeError:
+      try:
+        roic_avgs.append(self.stockrow_key_stats.roic_averages[1])
+      except IndexError:
+        return roic_avgs
+    try:
+      roic_avgs.append(self.stockrow_key_stats.roic_averages[2])
+      roic_avgs.append(self.stockrow_key_stats.roic_averages[-1])
```

Comment:
Do we need to check to make sure the data has a max value? In other words, is it possible for the `-1` index to be the same as the `2` index? If the `roic_averages` only has 3 values in some cases?

Reply:
Changed from -1 to 3.